src/flexmeasures_client/client.py

```diff
@@ -436,3 +436,51 @@ async def get_sensor_data(
         data_fields = ("values", "start", "duration", "unit")
         sensor_data = {k: v for k, v in response.items() if k in data_fields}
         return sensor_data
+
+    async def get_sensor(self, sensor_id: int):
+        uri = f"sensors/{sensor_id}"
+        response, status = await self.request(uri=uri, method="GET")
+        check_for_status(status, 200)
+        return response
+
+    async def add_sensor(
+        self,
+        name: str,
+        event_resolution: str,
+        unit: str,
+        generic_asset_id: int,
+        **kwargs,
+    ):
+        sensor = dict(
+            name=name,
+            event_resolution=event_resolution,
+            unit=unit,
+            generic_asset_id=generic_asset_id,
+            **kwargs,
+        )
+        uri = "sensors"
+        response, status = await self.request(uri=uri, json=sensor, method="POST")
+        check_for_status(status, 201)
+        return response
+
+    async def add_asset(
+        self, name, account_id, latitude, longitude, generic_asset_type_id
```

Comment:
I would add type hints in the input arguments and the return (just `dict` I think).

Reply:
Added type hints and docstrings to the functions